modules/nf-httpfs/src/main/nextflow/file/http/LatchFileSystemProvider.groovy

```diff
@@ -237,7 +237,15 @@ class LatchFileSystemProvider extends XFileSystemProvider {
     void delete(Path path) throws IOException {}
 
     @Override
-    void copy(Path source, Path target, CopyOption... options) throws IOException {}
+    void copy(Path source, Path target, CopyOption... options) throws IOException {
+        if (target.scheme == "latch" && source.scheme == "file") {
+            LatchPath lp = (LatchPath) target
+            lp.upload(source)
```

Comment:
its probably fine but are we good to ignore the copy options? Like what happens if the file exists on the remote?

Reply:
ill add a todo to check and add good errors for that + other failure modes like directory exists at path, etc